stable/ipfs/templates/statefulset.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apps/v1beta1
+kind: StatefulSet
+metadata:
+  name: {{ template "name" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+spec:
+  replicas: {{ .Values.replicaCount }}
+  serviceName: {{ template "name" . }}
+  volumeClaimTemplates:
```

Comment:
I could if it is a requirement for merging :) but it's not particularly useful without persistent volumes however - but I guess so is the case for MongoDB!